README.md

```diff
@@ -1,72 +1,72 @@
-# CMSIS Driver Validation
+# CMSIS-Driver Validation
 
-The branch *master* of this GitHub repository contains the CMSIS-Driver Validation. The [documentation](http://arm-software.github.io/CMSIS-Driver_Validation/html/index.html) is available under 
-http://arm-software.github.io/CMSIS-Driver_Validation/html/index.html
+Test suite for verifying that a peripheral driver implementation is compliant with the correposponding [CMSIS-Driver Specification](https://arm-software.github.io/CMSIS_5/Driver/html/index.html).
 
-Use [Issues](https://github.com/ARM-software/CMSIS-Driver_Validation#issues-and-labels) to provide feedback and report problems for CMSIS-Driver Validation.
+## Overview
+The branch *main* of this repository contains the code of CMSIS-Driver Validation Suite/[User documentation](http://arm-software.github.io/CMSIS-Driver_Validation/main/index.html) explains the scope and the usage of the framework.
 
-**Note:** The branch *develop* of this GitHub repository reflects our current state of development and is constantly updated. It gives our users and partners contiguous access to the CMSIS development. It allows you to review the work and provide feedback or create pull requests for contributions.
+[See verified releases](https://github.com/ARM-software/CMSIS-Driver_Validation/releases) of CMSIS-Driver Validation suite in source code archives and in [CMSIS-Pack (.pack)](https://open-cmsis-pack.github.io/Open-CMSIS-Pack-Spec/main/html/index.html) format as download assets.
 
-## Directory Structure
+Feel free to raise an [issue on GitHub](https://github.com/ARM-software/CMSIS-Driver_Validation/issues) to report a misbehavior (i.e. bugs), start discussions about enhancements or create a pull request with suggested modifications.
 
-| Directory        | Content                                                   |
-| ---------------- | --------------------------------------------------------- |
-| Boards           | Driver Validation examples for a set of boards            |
-| DoxyGen          | Source of the documentation                               |
-| Include          | Header files for driver validation components             |
-| Source           | Source files for driver validation components             |
-| Tools            | SockServer and SPI_Server source and executable files     |
-| Utilities        | Utility programs                                          |
+### Supported Driver Interfaces
 
-## Generate CMSIS Pack for Release
+The CMSIS-Driver Validation provides tests for the following CMSIS-Driver interfaces:
```

Comment:
I would update this section as follows:

The CMSIS-Driver Validation provides tests for the following CMSIS-Driver interfaces:

Extensive tests (with available test servers):
- **SPI**        - Serial Peripheral Interface driver.
- **USART**      - Universal Synchronous and Asynchronous Receiver/Transmitter interface driver.
- **WiFi**       - Wireless Fidelity Interface module/shield driver.

Basic tests:  
- **Ethernet**   - Ethernet MAC and PHY peripheral interface driver.
- **I2C**        - Inter-Integrated Circuit multi-master serial single-ended bus interface - driver.
- **MCI**        - Memory Card Interface driver for SD/MMC memory.
- **USB Device** - Universal Serial Bus Device interface driver.
- **USB Host**   - Universal Serial Bus Host interface driver.

Reply:
Thanks, will update